.goreleaser.yaml

```diff
@@ -228,6 +228,8 @@ dockers:
       - "--target=default"
     image_templates:
       - "ghcr.io/openbao/openbao:{{ .Version }}-alpine-amd64"
+      - "quay.io/openbao/openbao:{{ .Version }}-alpine-amd64"
+      - "hub.docker.com/openbao/openbao:{{ .Version }}-alpine-amd64"
```

Comment:
`hub.docker.com` should be omitted everywhere while building.

Their registry is called `docker.io` but it should work without it.

Reply:
Also for login, all image templates/manifests in the goreleaser file.

Reply:
Also would omitting the name cause confusing in the future?

Reply:
I usually omit it, but whatever you feel like is best. I am also not a maintainer. 🫣

Reply:
@cipherboy what do you recommend? I left it as docker.io for now.